docs/src/operator/usage.md

```diff
@@ -32,6 +34,59 @@ miden-node bundled bootstrap \
   --accounts-directory .
 ```
 
+You can also configure the account and asset data in the genesis block by passing in a toml configuration file.
+This is particularly useful for setting up test scenarios without requiring multiple rounds of
+transactions to achieve the desired state. Any account secrets will be written to disk inside the
+the provided `--accounts-directory` path in the process.
+
+```sh
+miden-node bundled bootstrap \
+  --data-directory data \
+  --accounts-directory . \
+  --genesis-config-file genesis.toml
+```
+
+The genesis configuration file should contain at least one faucet, and optionally, wallet definitions
+with assets, for example:
+
+```toml
+# The timestamp associated to the genesis block.
+# It will influence the hash of the genesis block.
+timestamp = 1717344256
+# Defines the format of the block protocol to use
+# for the genesis block.
+version   = 1
```

Comment:
nit: why do we wrap comments so early? Should we do at least 60 or maybe even 80 chars per line?